policyengine_us/variables/gov/states/co/tax/income/credits/sales_tax_refund/co_sales_tax_refund_eligible.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class co_sales_tax_refund_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "Eligible for the Colorado sales tax refund"
+    definition_period = YEAR
+    reference = "https://tax.colorado.gov/sites/tax/files/documents/DR_0104_Book_2022.pdf#page=23"
+    defined_for = StateCode.CO
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.co.tax.income.credits.sales_tax_refund
+        age_head = tax_unit("age_head", period)
+        head_eligible = age_head >= p.age_threshold
+        age_spouse = tax_unit("age_spouse", period)
+        spouse_eligible = age_spouse >= p.age_threshold
+        income_tax = tax_unit(
+            "co_income_tax_before_non_refundable_credits", period
+        )
+        filing_status = tax_unit("filing_status", period)
+        joint = filing_status == filing_status.possible_values.JOINT
+        age_eligible = where(
+            joint, head_eligible & spouse_eligible, head_eligible
```

Comment:
Source on both spouses needing to be 18+? I didn't see that in the form or law.

Reply:
I read this as both spouses have to meet the eligibility 

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/464b9e7b-53a5-47d5-bb0f-d81ac4f147fa)